src/main/scala/firrtl/LoweringCompilers.scala

```diff
@@ -132,7 +134,7 @@ import firrtl.transforms.BlackBoxSourceHelper
   */
 class NoneCompiler extends Compiler {
   def emitter = new ChirrtlEmitter
-  def transforms: Seq[Transform] = Seq.empty
+  lazy val transforms: Seq[Transform] = Seq(new IdentityTransform(ChirrtlForm))
```

Comment:
Why `lazy val`? It doesn't really matter either way I guess

Reply:
The require I added is eager, so this is either a def or a lazy val. It seemed better to be a val to prevent reconstructing the transform sequence everytime with a def.

Removing the require and doing `::` could make this a val again.

Reply:
This is the odd man out here, changing back to def.